pkg/api/folder.go

```diff
@@ -84,8 +86,22 @@ func UpdateFolder(c *models.ReqContext, cmd models.UpdateFolderCommand) response
 	return response.JSON(200, toFolderDto(g, cmd.Result))
 }
 
-func DeleteFolder(c *models.ReqContext) response.Response {
+func (hs *HTTPServer) DeleteFolder(c *models.ReqContext) response.Response { // temporary adding this function to HTTPServer, will be removed from HTTPServer when librarypanels featuretoggle is removed
```

Comment:
Is there any reason not to keep it as a method of HTTPServer after feature toggle is gone? I don't see the harm personally, there's often a need to use state on `HTTPServer`.
```suggestion
func (hs *HTTPServer) DeleteFolder(c *models.ReqContext) response.Response { // temporarily adding this function to HTTPServer, will be removed from HTTPServer when librarypanels featuretoggle is removed
```

Reply:
I think after feature toggle is gone this will be part of delete here https://github.com/grafana/grafana/blob/ad37077610f36bea8eb904edb9b519f1991baaed/pkg/services/sqlstore/dashboard.go#L388